_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -992,16 +992,26 @@ Set the variables for the `account-baseline-root` module in this environment in
 .infrastructure-live/root/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
+locals {
+  accounts = {
+    root = "216044045972"
+  }
```

Comment:
NIT: Perhaps this ID could be stored as a `root` key in the `accounts.json` that I mention in the comment under the `logs` section below.